### PR TITLE
Header improvements

### LINKED ISSUE #41: Add minimize/maximize window controls
without destroying design, that'd be great.

### LINKED ISSUE #305: Cover Component
In order to achieve https://github.com/KeitIG/museeks/issues/130, we should create a `Cover` component.

- Just with a string parameter with the folder to scan.
- Remove cover from the store

### LINKED ISSUE #311: Ctrl-F focuses the search input
All in title

### LINKED ISSUE #328: Move Queue into a Bootstrap Dropdown
So it disappears on blur.

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #41 Min-max window controls 
Fix #305 Cover component
Fix #311 Ctrl-F focuses search
Fix #328 comma-dangle eslint rule

Still a layout problem I'll try to fix asap

### CODE REVIEW

**Comment by KeitIG** [comment]
#yolo

File: src/styles/_header.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
+$_header-color: #555;
```
Comment (YurySolovyov) [inline src/styles/_header.scss]: why leading underscore?

Comment (KeitIG) [inline src/styles/_header.scss]: because they're local variables

**Comment by YurySolovyov** [comment]
> Still a layout problem I'll try to fix asap

Which one?

**Comment by KeitIG** [comment]
Volume button is overlapping on the cover

**Pull request merged**